fs/fs.go

```diff
@@ -439,7 +439,7 @@ func (i *RealFsInfo) GetFsInfoForPath(mountSet map[string]struct{}) ([]Fs, error
 
 var partitionRegex = regexp.MustCompile(`^(?:(?:s|v|xv)d[a-z]+\d*|dm-\d+)$`)
 
-func getDiskStatsMap(diskStatsFile string) (map[string]DiskStats, error) {
+func GetDiskStatsMap(diskStatsFile string) (map[string]DiskStats, error) {
```

Comment:
Do you think it would be possible to use `RealFsInfo.GetFsInfoForPath()` rather than exporting yet another function? It would be great to limit public interfaces as much as possible.

Reply:
It was indeed possible and I have made getDiskStatsMap private again in my latest commit.